docs/en/observability/slo-create.asciidoc

```diff
@@ -0,0 +1,102 @@
+[[slo-create]]
+= Create an SLO 
+
+beta::[]
+
+To create and SLO, go to *Observability → SLOs*: 
+* If you're creating your first SLO, you'll see an introductory page. Click the *Create SLO* button.
+* If you have previously created SLOs, click the *Create new SLO* button in the upper-right corner of the SLO page.
+
+To create an SLO, you need to:
+
+. Define your service level indicator (SLI)
+. Set your objectives
+. Describe your SLO
+
+[discrete]
+[[define-sli]]
+== Define your SLI
+//Maybe we can do more here, why/when you should use each of these?
+
+You can choose one of the following SLI types:
+
+[horizontal]
+*Custom KQL*:: This indicator is based on any elasticsearch index or index pattern you have. You simply define two queries, one that yields the good events and one that yields the total events from your index.
+*APM latency*:: This indicator is based on the APM data that we receive from your instrumented services. We need the service name and the latency threshold at a minimum, but you can also provide an environment, transaction type, transaction name and a custom filter on the APM metrics data. 
+*APM availability*:: This indicator is based on the APM data that we receive from your instrumented services. We need the service name and the list of good status codes at a minimum, but you can also provide an environment, transaction type, transaction name and a custom filter on the APM metrics data. 
+
+[discrete]
+[[custom-kql-sli]]
+=== Custom KQL
+When defining a custom KQL SLI, you need to set the following fields:
+
+* *Index* —
+* *Timestamp field* —
+* *Query filter* —
+* *Good query* —
+* *Total query* —
+
+[discrete]
+[[apm-latency-sli]]
+//This can probably be combined with the APM availability section since the only difference are the Threshold and Good status codes fields.
+
+=== APM latency
+When defining an APM latency SLI, you need to set the following fields:
+
+* *Service name* —
+* *Service environment* —
+* *Transaction type* —
+* *Transaction name* —
+* *Threshold* —
+* *Query filter* —
+
+[discrete]
+[[apm-availability-sli]]
+=== APM availability
+When defining an APM availability SLI, you need to set the following fields:
+
+* *Service name* —
+* *Service environment* —
+* *Transaction type* —
+* *Transaction name* —
+* *Good status codes* —
+* *Query filter* —
+
+[discrete]
+[[set-slo]]
+== Set your objectives
+After defining your SLI, you need to set your objectives. 
+
+[discrete]
+[[slo-budgeting-method]]
+=== Budgeting method
+//can we be more specific here? Equations used possibly?
+You can select either an *occurrences* or a *timeslices* budgeting method: 
+
+[horizontal]
+*Occurences*:: Uses the number of good events and the number of total events to compute the SLO.
```

Comment:
```suggestion
*Occurrences*:: Uses the number of good events and the number of total events to compute the SLO.
```

Reply:
Here's as well we can add an example:
Let's take a 30 days rolling SLO with a 95% target, and over the past 30 days we got `1,355,700` total events. The error budget is `100-95 = 5%`, or about 66,785 bad events tolerated before violating the SLO.

If we had 1,300,000 good events over the same period, the observed value is `Good Events / Total Events = 0.95891421 => 95.89%`